data_safe_haven/utility/console.py

```diff
@@ -0,0 +1,25 @@
+from rich import print as rprint
```

Comment:
As this is still to do with printing information to screen, I think it could/should live in the logging module. Happy if you think that means we need to change the module name to something broader.

Reply:
I could see a module for "console stuff" like formatting text and prompts.

I think that is quite different from logging though and we should keep them separate.

I will leave it as is for this PR.